meson.build

```diff
@@ -0,0 +1,93 @@
+project('purple-facebook', 'c')
+
+# Define package version
+#PACKAGE_VERSION = '0.9.61'  # Adjust the version number as needed
+#PACKAGE_URL = 'https://github.com/DMJC/purple-facebook'
+
+# Configure the package
+#pkgconf = configuration_data()
+#pkgconf.set('PACKAGE_VERSION', PACKAGE_VERSION)
+#pkgconf.set('PACKAGE_URL', PACKAGE_URL)
+
+compiler_options = []
+compiler_options += ['-DPACKAGE_VERSION="0.9.7"']
+compiler_options += ['-DPACKAGE_URL="https://github.com/DMJC/purple-facebook"']
```

Comment:
Will the url need to be updated after the pr is merged, or would you change as part of this pr?

Reply:
It'll change as part of the PR, but it can be edited during the PR afaik. I wasn't sure if the project was maintained or not.

Reply:
I've done a change to the debian files in my repository, the changes now build a working debian package using the Meson build system.

Reply:
I've updated the URL in the file to have the dequis repository.